docs/en/observability/logs-checklist.asciidoc

```diff
@@ -2,78 +2,89 @@
 = Logs resource guide
 
 ++++
-<titleabbrev>Resource guide</titleabbrev>
+<titleabbrev>Logs resource guide</titleabbrev>
 ++++
 
-In this guide, you'll find resources on sending log data to {es}, configuring your logs, and analyzing your logs.
+Logs are an important tool for ensuring the performance and reliability of your applications and infrastructure. They provide important information for debugging, analyzing performance, and managing compliance.
+
+In this guide, you'll find resources for sending log data to {es}, configuring your logs, and analyzing your logs.
 
 [discrete]
 [[logs-getting-started-checklist]]
 == Get started with logs
 
-If you're new to ingesting, viewing, and analyzing logs with Elastic, see <<logs-metrics-get-started, Get started with logs and metrics>> for an overview of adding integrations, installing and running an {agent}, and monitoring logs.
+For a high-level overview on ingesting, viewing, and analyzing logs with Elastic, refer to <<logs-metrics-get-started, Get started with logs and metrics>>.
+
+To get started ingesting, parsing, and filtering your own data, refer to these pages:
+
+* *<<logs-stream>>*: send log files from your system to {es} using a standalone {agent} and configure the {agent} and your data streams using the `elastic-agent.yml` file.
+* *<<logs-parse>>*: break your log messages into meaningful fields that you can use to filter and analyze your data.
+* *<<logs-filter>>*: find specific information in your log data to gain insight and monitor your systems.
+
+The following sections in this guide provide resources to important concepts or advanced use cases for working with your logs.
 
 [discrete]
 [[logs-send-data-checklist]]
 == Send logs data to {es}
 
-You can send logs data to {es} in different ways depending on your needs:
+You can send log data to {es} in different ways depending on your needs:
 
-- {agent}
-- {filebeat}
+* *{agent}*: a single agent for logs, metrics, security data, and threat prevention. It can be deployed either standalone or managed by {fleet}:
```

Comment:
```suggestion
* *{agent}*: a single agent for logs, metrics, security data, and threat prevention. It can be deployed either standalone or managed by {fleet}.
```

Reply:
ohhhh nvm, I see